docs/gallery/advanced_io/h5dataio.py

```diff
@@ -234,7 +234,7 @@
 
 wrapped_data = H5DataIO(
     data=data,
-    compression=hdf5plugin.Zstd().filter_id,
+    **hdf5plugin.Zstd().filter_id,  # set the compression and compression_opts parameters
```

Comment:
```suggestion
    **hdf5plugin.Zstd(clevel=3),  # set the compression and compression_opts parameters
```
The current diff with the gets the attribute `filter_id` so it is wrong.
I think having the options over there explicitly also helps the users to understand that this is the way that the options are meant to be passed. At least it would have helped me.

Reply:
Thanks for the fix